fetch.bs

```diff
@@ -1121,6 +1109,46 @@ or "<code>worker</code>".
  <li><p>Return <var>newRequest</var>.
 </ol>
 
+<p>To <dfn export for=request id=concept-request-upload-body>upload body</dfn> of a
```

Comment:
I think we should avoid "upload" entirely. The protocol specification doesn't use it either and it doesn't make much sense for non-HTTP URLs (although I guess those will fail to function anyway).